inc/rest-api/relationship/class-posts-controller.php

```diff
@@ -213,9 +213,10 @@ public function get_items( $request ) {
 		$result_posts = [];
 		foreach ( $results->posts as $result ) {
 			$result_posts[] = array(
-				'id'    => $result->ID,
-				'value' => $result,
-				'label' => 'draft' === $result->post_status ? sprintf( '%s %s', $result->post_title, '(Draft)' ) : $result->post_title,
+				'id'        => $result->ID,
+				'value'     => $result,
+				'label'     => 'draft' === $result->post_status ? sprintf( '%s %s', $result->post_title, '(Draft)' ) : $result->post_title,
+				'permalink' => $this->get_permalink_with_dynamic_support( $result->ID ),
```

Comment:
@KhokanTP Let's use the `gumponents_posts_relationship_results` hook for this in the project